packages/ui/components/ui/tx/view/asset-icon/delegation-token-icon.tsx

```diff
@@ -0,0 +1,105 @@
+import { assetPatterns } from '@penumbra-zone/constants/src/assets';
+
+const getFirstEightCharactersOfValidatorId = (displayDenom = ''): [string, string] => {
+  const id = (assetPatterns.delegationToken.capture(displayDenom)?.id ?? '').substring(0, 8);
+
+  const firstFour = id.substring(0, 4);
+  const lastFour = id.substring(4);
+
+  return [firstFour, lastFour];
+};
+
+export const DelegationTokenIcon = ({
```

Comment:
![image](https://github.com/penumbra-zone/web/assets/1121544/4ac780c6-1d1e-4e09-9518-9f49ded689a7)

Reply:
Noticed that the validator "Penumbra Labs CI 1/2" doesn't have their icons in the registry rendering. Can you update this: https://github.com/penumbra-zone/web/blob/main/packages/constants/src/local-asset-registry.json#L123-L174

Reply:
@grod220 looks like the token in the asset registry doesn't match the actual CI delegation token:

Penumbra Labs CI 1 in registry: `delegation_penumbravalid18nkv0r3sfp2seleq6du5kt3mhfce3k6cqm77kj2e7mhakmyw9v9qx42a20`
Penumbra Labs CI 1 from the testnet RPC node: `penumbravalid1dj98v5xvm63xv9tuazeggrjjac668yeam2yryp5duhdawywwuvxq2du5a7`

Reply:
Yeah, our registry is wrong. Can you update that? 

and also make a PR here too?  https://github.com/prax-wallet/registry/

Reply:
Done:
![image](https://github.com/penumbra-zone/web/assets/1121544/849d419e-2dbe-4d4b-942d-0458d9e54b2a)

Will do the prax-wallet one next

Reply:
Looks like the prax-wallet registry is already up to date!